File: AndroidVersion4.1.3/Budget App/app/src/main/java/com/example/budgy/Models/DailyTransactions.java
package com.example.budgy.Models;

import android.net.Uri;

import java.net.URI;

public class DailyTransactions {
    int id;
    String category;
    Double amount;
    String type;
    int date;
    String receipt;
    int userId;
    String title;

    public DailyTransactions(int id, String category, Double amount, String type, int date, String receipt,String title, int userId) {
        this.id = id;
        this.category = category;
        this.amount = amount;
        this.type = type;
        this.date = date;
        this.receipt = receipt;
        this.userId = userId;
        this.title = title;
    }

    public DailyTransactions() {

    }

    public int getId() { return id;}

    public void setId(int id) { this.id = id;}

    public String getCategory() {   return category;}

    public void setCategory(String category) {  this.category = category;}

    public Double getAmount() { return amount;}

    public void setAmount(Double amount) {  this.amount = amount;}

    public String getType() {   return type;}

    public void setType(String type) {  this.type = type;}

    public int getDate() { return date;}

    public void setDate(int date) {  this.date = date;}

    public String getReceipt() { return receipt;}

    public void setReceipt(String receipt) {  this.receipt = receipt;}

    public String getTitle() { return title; }

    public void setTitle(String title) { this.title = title; }

    public int getUserId() {  return userId; }

    public void setUserId(int userId) {  this.userId = userId; }
}

File: 2021 Special Topic Web Mobile App/PocketBook_Version2/app/src/main/java/com/example/pocketbook/entities/DailyTransactions.kt
package com.example.pocketbook.entities

class DailyTransactions {
    var id = 0
    var category: String? = null
    var amount: Double? = null
    var type: String? = null
    var date = 0
    var receipt: String? = null
    var userId = 0
    var title: String? = null

    constructor(
        id: Int,
        category: String?,
        amount: Double?,
        type: String?,
        date: Int,
        receipt: String?,
        title: String?,
        userId: Int
    ) {
        this.id = id
        this.category = category
        this.amount = amount
        this.type = type
        this.date = date
        this.receipt = receipt
        this.userId = userId
        this.title = title
    }

    constructor() {}
}